### ISSUE TITLE
Data Augmentation and loss functions 

### ISSUE THREAD

**Issue by waquey** [deepinsight/insightface#718]
Hello, I'd like to ask some questions about RetinaFace 

(1) Paper illustrates training data would apply color distortion or random crop to do  data augmentation. 

I've seen "rcnn.core.loader" Does this implement random crop or which file can I check to do "random crop" and "color distortion" 
 
(2) Landmark loss used L1 loss, if we want to use L2 or smoothL1 instead, we can add "class RPN_ooo_Metric(mx.metric.EvalMetric):" to implement?
